assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -543,6 +545,25 @@ const baseActions = {
 			dispatch( MODULES_ANALYTICS_4 ).saveSettings();
 		}
 	},
+
+	/**
+	 * Sets if GA4 has mismatched Google Tag ID.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {boolean} hasMismatchedTag If GA4 has mismatched Google Tag.
+	 * @return {Object} Redux-style action.
+	 */
+	*setHasMismatchedGoogleTagID( hasMismatchedTag ) {
+		if ( ! isFeatureEnabled( 'gteSupport' ) ) {
+			return;
+		}
```

Comment:
Is this necessary? We primarily check for features to conditionally call an action rather than baking these checks into the store itself.

Reply:
Understood, removed, thank you Evan!